generators/server/templates/src/main/resources/config/liquibase/changelog/initial_schema.xml.ejs

```diff
@@ -163,15 +163,24 @@
                   <%_ if (devDatabaseType === 'mssql' || prodDatabaseType === 'mssql') { _%>
                   identityInsertEnabled="true"
                   <%_ } _%>
-                  tableName="<%= jhiTablePrefix %>_authority"/>
+                  tableName="<%= jhiTablePrefix %>_authority">
+            <%_ if (devDatabaseType === 'oracle' || prodDatabaseType == 'oracle') { _%>
+            <column name="name" type="string"/>
+            <%_ } _%>
+        </loadData>
 
         <<% if (devDatabaseType === 'mssql' || prodDatabaseType === 'mssql') { %>ext:<% } %>loadData
                   file="config/liquibase/data/user_authority.csv"
                   separator=";"
                   <%_ if (devDatabaseType === 'mssql' || prodDatabaseType === 'mssql') { _%>
                   identityInsertEnabled="true"
                   <%_ } _%>
-                  tableName="<%= jhiTablePrefix %>_user_authority"/>
+                  tableName="<%= jhiTablePrefix %>_user_authority">
+            <%_ if (devDatabaseType === 'oracle' || prodDatabaseType === 'oracle') { _%>
```

Comment:
I wonder: maybe we can specify the columns regardless of the databaseType

Reply:
Yes, we can check whether it works for all dbs or not. We can rely on CI results too I think.